### ISSUE TITLE
Incorrect heading is used (starts with h3 instead of h1)

### ISSUE THREAD

**Issue by emma-sg** [emma-sg/roleypoly#9]
The first heading on a webpage should be an `h1`, yet it is an `h3` in this case.
